homeassistant/components/twinkly/diagnostics.py

```diff
@@ -33,7 +33,8 @@ async def async_get_config_entry_diagnostics(
     return async_redact_data(
         {
             "entry": entry.as_dict(),
-            "device_info": hass.data[DOMAIN][entry.entry_id][DATA_DEVICE_INFO],
+            "device_info": hass.data[DOMAIN][entry.entry_id][DATA_DEVICE_INFO]
+            | {ATTR_SW_VERSION: hass.data[DOMAIN][entry.entry_id][ATTR_SW_VERSION]},
```

Comment:
I added it back to the device_info in diagnostics as it sort of belong there, but if you prefer, I can move it to its own attribute.

Reply:
Why would you mangle this output?

Basically you reverted a review comment from above...?

Reply:
I was not sure if it was only the internal representation of device_info or the diagnostics output you were concerned about.
Fixed now